src/hooks/useENSRegistrationForm.ts

```diff
@@ -205,7 +223,7 @@ export default function useENSRegistrationForm({
   useEffect(() => {
     if (mode === REGISTRATION_MODES.EDIT) {
       if (profileQuery.isSuccess || !isEmpty(values)) {
-        setTimeout(() => setIsLoading(false), 200);
```

Comment:
Think this was here to avoid a keyboard FPS drop. Is this still happening?

Reply:
i'm gonna add it back then, just wanted to reduce that time